### PR TITLE
Add ability to disable popover menu items

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
* **#479** <a href="https://app.graphite.dev/github/pr/terraware/web-components/479?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nickgraz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/PopoverMenu/Popover.tsx [hunk 1]
```diff
@@ -72,12 +72,7 @@ export default function PopoverDropdown(props: PopoverProps): JSX.Element {
             ...elements,
             ...section.map((item, itemIndex) => {
               return (
-                <MenuItem
-                  onClick={() => onClick(item)}
-                  key={`option-${itemIndex}`}
-                  sx={itemStyles(item.type)}
-                  disableRipple={true}
-                >
+                <MenuItem onClick={() => onClick(item)} key={`option-${itemIndex}`} sx={itemStyles(item.type)} disableRipple={true} disabled={item.disabled}>
```
Comment (nickgraz) [inline src/components/PopoverMenu/Popover.tsx:75]: The prettier `printWidth` in this repo is 160, should I update it to 120 to match `terraware-web`?

**Comment by nickgraz** [comment]
publish rc

**Review by karthikbtf — APPROVED** [review]
Nice 🎉 
Didn't know this feature was missing.

File: src/components/PopoverMenu/Popover.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/components/PopoverMenu/Popover.tsx:75]: Being consistent sounds good.

Comment (nickgraz) [reply src/components/PopoverMenu/Popover.tsx]: Cool, `prettier` isn't used in the build flow so I did not opt for a "apply new prettier config to every file" commit. Let me know if you'd like to see that in another PR or if you are OK that it is slowly applied to files that are touched in the future.

Comment (karthikbtf) [reply src/components/PopoverMenu/Popover.tsx]: Fine with what you suggest. In the interest of time, I agree with the as-touched in the future approach.

**Pull request merged**